03.bowling/bowling.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  shots << if s == 'X'
+             10
+           else
+             s.to_i
+           end
+end
+
+frame = 0
+max_frame = 10
+count = 0
```

Comment:
countっていう変数名だと何をカウントしているのかわかりづらいです。
そもそもこの変数は一投目が終わったかどうか（もしくは次のフレームへ進むかどうか）を管理するフラグという扱いなのかなと思います。

命名がちょっと難しいですが、

```suggestion
first_done = false
```

みたいにbooleanのフラグにして、「trueになっていたら次のframeに進む」みたいな管理するのはどうかなと思いました